static/js/publisher/release/releasesController.js

```diff
@@ -461,6 +490,20 @@ export default class ReleasesController extends Component {
       }
     );
 
+    let filteredRevisions = this.props.revisions;
+    let title = null;
+    let filters = this.state.revisionsFilters;
+
+    if (filters && filters.arch) {
+      title = "Latest revisions";
```

Comment:
Could this be the initialisation value at line 494?

Reply:
No, if there are no filters we want title to be `null`, so `RevisionsList` will use its default value 'Available revisions'. But I'm thinking about removing title 'logic' from `RevisionsList` altogether, so it will hopefully make more sense in some of the next iterations of this code.